telegram/src/main/kotlin/me/ivmg/telegram/network/ApiClient.kt

```diff
@@ -51,17 +52,18 @@ private fun convertFile(name: String, file: SystemFile, mimeType: String? = null
 private fun convertBytes(
     name: String,
     bytes: ByteArray,
-    mimeType: String? = null
+    mimeType: String? = null,
+    filename: String? = null
 ): MultipartBody.Part {
     val mediaType = mimeType?.let { MediaType.parse(it) }
     val requestBody = RequestBody.create(mediaType, bytes)
 
-    return MultipartBody.Part.createFormData(name, name, requestBody)
+    return MultipartBody.Part.createFormData(name, filename ?: "untitled", requestBody)
```

Comment:
I would make `filename` mandatory in the new `sendDocument` method added to support `ByteArray` and wouldn't add "untitled" as default value for it on `convertBytes` method. I think this could be confusing for someone using the mentioned method and that's better to make mandatory indicating the filename in that case. If user wants to have "untitled" as title they still can, but will be totally under their responsibility. What do you think about it?

Reply:
Hello! @vjgarciag96 I think mandatory `filename` parameter might be more clear for usage. Do you want me to refactor my code or you will do it by your self?

Reply:
As you want, I can do the changes but you'll probably have to wait to the weekend. If you're ok with that, I can take care of it.